.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: [20.x, 19.x, 18.x, 16.x, 14.x]
+        node: [21.x, 20.x, 19.x, 18.x, 16.x, 14.x]
         include:
         - os: windows-latest
           node: "18.x"
```

Comment:
In this PR or a separate PR, we can also change Node 18 to Node 20 in a few places.

Reply:
Makes sense since 20 is the current LTS version now and 18 is in maintenance mode.